src/sections/services.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div
+      class="font-[outfit] flex flex-col gap-24 lg:gap-0 lg:flex-row justify-around mt-12 bg-[#EFECEA] px-8 sm:px-28 lg:px-52"
+    >
+      <ul
+        class="lg:text-5xl xl:text-6xl text-[#000000] font-[500] leading-[100%] lg:w-[563px] lg:h-[780px] flex flex-col lg:gap-24 text-3xl gap-12"
+      >
```

Comment:
1. No use of arbitrary values for colors.
2. Configure these large values and use custom utility classes. Arbitrary don't scale very well with consistency. 
```suggestion
      <ul
        class="lg:text-5xl xl:text-6xl text-black font-[500] leading-[100%] lg:w-large lg:h-large flex flex-col lg:gap-24 text-3xl gap-12"
      >
```